app/create_file.py

```diff
@@ -1 +1,48 @@
-# write your code here
+import os
+import sys
+from datetime import datetime
+
+
+def get_clean_command(cmd: str) -> str:
+    return " ".join(sys.argv).split(cmd)[1]
+
+
+def file_creation(file_path: str) -> None:
```

Comment:
```suggestion
def create_file(file_path: str) -> None:
```
Suggeesting this name, because it seems more appropriate, when you use it in other functions, like function created file, not simply calls for file creation... Just a suggestion here)

Reply:
ok, but won't it create confusion because of the same filename?